contentcuration/contentcuration/frontend/shared/views/BottomBar.vue

```diff
@@ -9,6 +9,13 @@
 
   export default {
     name: 'BottomBar',
+    props: {
+      // eslint-disable-next-line kolibri/vue-no-unused-properties
+      appearanceOverrides: {
+        type: Object,
+        default: () => ({ height: '64px' }),
```

Comment:
With the proposed change on the style bellow, this default value will not be required. And this eslint-disable line can be erased too :open_hands:

Reply:
@AlexVelezLl I have generalized appearanceOverrides